spec/lib/percy/capybara_spec.rb

```diff
@@ -76,4 +76,14 @@
       expect(Percy::Capybara.capybara_client.enabled?).to eq(false)
     end
   end
+  describe '#use_loader' do
+    class DummyLoader < Percy::Capybara::Loaders::NativeLoader; end
+
+    it 'sets the current capybara clients custom loader flag' do
+      capybara_client = Percy::Capybara::Client
+      expect(Percy::Capybara.capybara_client.custom_loader?).to eq(false)
+      Percy::Capybara.use_loader DummyLoader
```

Comment:
For style consistency with other things around, use parens here `use_loader(DummyLoader)`